README.md

```diff
@@ -101,19 +101,17 @@ This part is relevant only if you want to develop the repository or build it on
 Only needed the first time:
 
 ```
+git clone https://github.com/Kixunil/cryptoanarchy-deb-repo-builder
+cd cryptoanarchy-deb-repo-builder
 sudo apt-get install cargo npm ruby-mustache
-# Required version >= 0.1.1
+# Requires the latest debcrafter version
 cargo install --git https://github.com/Kixunil/debcrafter
 PATH=$PATH:~/.cargo/bin
 cargo install cfg_me
-make build-dep
+make BUILD_DIR=$PWD/build build-dep
+make
```

Comment:
same `BUILD_DIR` parameter has to be added to the other `make` too

Reply:
```suggestion
make BUILD_DIR=$PWD/build
```